gateway/tests/e2e/feedback.rs

```diff
@@ -702,8 +697,8 @@ async fn e2e_test_boolean_feedback() {
     let response_json = response.json::<Value>().await.unwrap();
     let inference_id = response_json.get("inference_id").unwrap().as_str().unwrap();
     let inference_id = Uuid::parse_str(inference_id).unwrap();
-    // Sleep for 1 second to allow ClickHouse to write
-    tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+
+    // No sleeping, we should throttle in the gateway
 
     let payload = json!({"inference_id": inference_id, "metric_name": "task_success", "value": true, "tags": {"key": tag_value, "key2": tag_value2}});
```

Comment:
For complete sanity check, let's add a check where it sleeps >cooldown first to make sure that code path is correct as well.

Reply:
kk I'll change one to do that